schemas/expconf/v0/experiment.json

```diff
@@ -71,11 +71,27 @@
         "entrypoint": {
             "type": [
                 "string",
+                "array",
                 "null"
             ],
             "checks": {
-                "entrypoint must be of the form \"module.submodule:ClassName\"": {
-                    "pattern": "^[a-zA-Z0-9_.]+:[a-zA-Z0-9_]+$"
+                "entrypoint must contain a trial class of the form \"module.submodule:ClassName\"": {
+                    "contains": {
+                        "pattern": "^[a-zA-Z0-9_.]+:[a-zA-Z0-9_]+$"
+                    }
+                }
+            },
+            "items": {
+                "type": "string"
+            },
+            "if": {
+                "type": "string"
+            },
+            "then": {
```

Comment:
there's an `"if"`/`"then"` in json schema?? whaaat??

Reply:
this is necessary because the array/ "contains" check will fail when it's a string type